modules/social_features/social_content_report/src/Routing/RouteSubscriber.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace Drupal\social_content_report\Routing;
+
+use Drupal\Core\Routing\RouteSubscriberBase;
+use Symfony\Component\Routing\RouteCollection;
+
+/**
+ * Listens to the dynamic route events.
+ */
+class RouteSubscriber extends RouteSubscriberBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function alterRoutes(RouteCollection $collection) {
+    // Always deny access to '/user/logout'.
```

Comment:
@OH17L This comment isn't accurate because we're not doing anything with access here. That's why I laughed and added a `?`. Removing the line is the simplest solution.